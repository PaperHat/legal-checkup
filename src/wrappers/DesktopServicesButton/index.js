import React from 'react'
import StyledDesktopButton from '@components/styled/DesktopButton'

export default function DesktopAboutButton () {
  return (
    <StyledDesktopButton
      onClick={() => {
        window.open('http://www.wclc.org.nz/our-services/', '_blank')
      }}
    >
      Community Law Services
    </StyledDesktopButton>
  )
}
