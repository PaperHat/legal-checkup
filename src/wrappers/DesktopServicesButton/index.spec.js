import React from 'react'
import { shallow } from 'enzyme'

import DesktopServicesButton from '.'

describe('components:wrappers:DesktopServicesButton', () => {
  it('renders the DesktopServicesButton and CSS properties properly', () => {
    expect(toJson(shallow(<DesktopServicesButton />))).toMatchSnapshot()
  })
})
