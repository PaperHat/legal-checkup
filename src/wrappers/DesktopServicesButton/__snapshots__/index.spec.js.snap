// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components:wrappers:DesktopServicesButton renders the DesktopServicesButton and CSS properties properly 1`] = `
<StyledDesktopButton
  onClick={[Function]}
>
  Community Law Services
</StyledDesktopButton>
`;
